### DISCUSSION TITLE
Is AGH losing its relevance

### DISCUSSION THREAD

**Discussion by fizsef** [AdguardTeam/AdGuardHome#7402] [General]
I have been using AGH on my router for a long time, but recently, in light of recent events, when everyone at home began to use their DNS and VPN on their device, I thought about the meaning of this.
The number of packets that go through AGH has dropped by an order of magnitude.

**Comment by Dynamic5912** [comment]
That's an issue you need to fix then.

**Comment by fizsef** [comment]
And how will you solve it? You will not be able to force the device to go through AGH if it uses its VPN.

**Reply by Dynamic5912** [reply]
Again - you need to fix that in your setup/router/firewall.

It's not something that can be fixed in AGH.

**Comment by fizsef** [comment]
You won't be able to force them. If a child uses, and knows how to do it, to install a VPN on his smartphone, then my AGH on the router becomes useless.

**Reply by Dynamic5912** [reply]
OK. Once again, not AdGuard's issue to fix.

**Comment by fizsef** [comment]
Well, yes, I'm not saying that this is an AGH problem, I'm just sharing my opinion. It is a pity that it cannot be used simultaneously with products such as ruantiblock, PBR, AWG on the router.

**Reply by Dynamic5912** [reply]
> Well, yes, I'm not saying that this is an AGH problem, I'm just sharing my opinion. It is a pity that it cannot be used simultaneously with products such as ruantiblock, PBR, AWG on the router.

That's a limitation on your hardware.

Try implanting a blocklist in AGH that blocks VPN and DOH/DOT domains and IP's.

**Comment by fizsef** [comment]
But this is not a solution, it will use a different VPN every time, and it is unrealistic to constantly monitor this.

**Reply by Dynamic5912** [reply]
Use a list that blocks all _known_ VPN's then.
